addons/project/static/src/css/project.css

```diff
@@ -4,6 +4,12 @@
   border: 1px solid var(--success);
 }
 
+.o_project_kanban .oe_kanban_align.badge {
+  background: inherit;
+  color: inherit;
+  border: 1px solid var(--success);
+}
+
```

Comment:
Instead of this diff, you could do this in the first line of this file:
```diff
- .o_kanban_project_tasks .oe_kanban_align.badge {
+ .o_kanban_project_tasks .oe_kanban_align.badge, .o_project_kanban .oe_kanban_align.badge {
```

```suggestion
```